src/pages/evm/nfts/NftViewer.vue

```diff
@@ -123,6 +123,19 @@ function toggleVideoPlay(playOnly?: boolean) {
                 :alt="imageAlt"
                 class="c-nft-viewer__image"
             >
+            <div
+                v-else-if="nftType === nftTypes.video"
+                class="c-nft-viewer__video-container"
+                tabindex="0"
+                role="preview"
+            ><video
+                ref="videoElement"
+                :controls="false"
+                :src="nft.videoSrc"
+                :poster="nft.imageSrcFull"
+                playsinline
+                class="c-nft-viewer__video"
+            ></video></div>
```

Comment:
This was needed to show the video preview because it was impossible to extract a real image from the video file.